viewPost.php

```diff
@@ -1,31 +1,41 @@
 <?php
 
 include('config/init.php');
+$comment=getAllComments($_GET['blogPostId']);
+$result=getAllCategories($_GET ['blogPostId']);
```

Comment:
Does this work? Maybe PHP is clever enough to figure this out, but I'd expect the space between `$_GET` and `['blogPostId']` to be a problem.